cmd/cluster/cluster.go

```diff
@@ -80,6 +81,7 @@ func NewCreateCommands() *cobra.Command {
 	cmd.PersistentFlags().DurationVar(&opts.Timeout, "timeout", opts.Timeout, "If the --wait flag is set, set the optional timeout to limit the waiting duration. The format is duration; e.g. 30s or 1h30m45s; 0 means no timeout; default = 0")
 	cmd.PersistentFlags().Var(&opts.NodeUpgradeType, "node-upgrade-type", "The NodePool upgrade strategy for how nodes should behave when upgraded. Supported options: Replace, InPlace")
 	cmd.PersistentFlags().Var(&opts.OLMCatalogPlacement, "olmCatalogPlacement", "The OLM Catalog Placement for the HostedCluster. Supported options: Management, Guest")
+	cmd.PersistentFlags().BoolVar(&opts.OLMDisableDefaultSources, "olmDisableDefaultSources", opts.OLMDisableDefaultSources, "Disables the OLM default catalog sources for the HostedCluster.")
```

Comment:
My preference is to follow the convention used in other flags with hyphen-separated names. Given that we already used camel case for the catalog placement one, I see 2 options:
1) Introduce a hyphen-separated version of the catalog placement one (and deprecate the existing one) and make this one hyphen separated one.
2) Leave this one as camel case and open a ticket to update both flags in the future.